src/DynamoCoreWpf/Controls/IncanvasSearchControl.xaml

```diff
@@ -106,29 +112,24 @@
                 Grid.Row="1"
                 Name="FoundMembers"
                 MaxHeight="250">
-            <ScrollViewer>
-                <ListBox Name="MembersListBox"
-                         Background="Transparent"
-                         BorderBrush="Transparent"
-                         ItemContainerStyle="{StaticResource SearchMemberStyle}"
-                         ItemsSource="{Binding SearchResults, NotifyOnTargetUpdated=True}"
-                         TargetUpdated="OnMembersListBoxUpdated">
-                    <ListBox.ItemTemplate>
-                        <DataTemplate>
-                            <ScrollViewer Template="{StaticResource InCanvasScrollViewerControlTemplate}"
-                                          Style="{StaticResource InCanvasScrollViewerStyle}"
-                                          CanContentScroll="True">
-                                <ContentControl Template="{StaticResource MemberControlTemplate}" />
-                            </ScrollViewer>
-                        </DataTemplate>
-                    </ListBox.ItemTemplate>
-                    <ListBox.Template>
-                        <ControlTemplate>
-                            <ItemsPresenter />
-                        </ControlTemplate>
-                    </ListBox.Template>
-                </ListBox>
-            </ScrollViewer>
+            <ListBox Name="MembersListBox"
+                     Background="Transparent"
+                     BorderBrush="Transparent"
+                     ItemContainerStyle="{StaticResource SearchMemberStyle}"
+                     ItemsSource="{Binding SearchResults, NotifyOnTargetUpdated=True}"
+                     TargetUpdated="OnMembersListBoxUpdated"
+                     PreviewMouseWheel="OnMembersListBoxMouseWheel">
+                <ListBox.ItemTemplate>
+                    <DataTemplate>
+                        <ScrollViewer Template="{StaticResource InCanvasScrollViewerControlTemplate}"
+                                      Style="{StaticResource InCanvasScrollViewerStyle}"
+                                      CanContentScroll="True"
+                                      Width="250">
```

Comment:
Should the `ScrollViewer` width be tied to the `MembersListBox.Width`? Or is this by design?